### ISSUE TITLE
Roadmap/plan for relation to CDK

### ISSUE THREAD

**Issue by koliyo** [pulumi/pulumi-aws#1171]
The Amazon AWS CDK seem to provide a similar role to Pulumi. It is however clearly limited to the AWS context.

But I think it is important to understand if for example these could be compatible. Eg allow pulumi to load/import a CDK declaration for instance?

For instance CDK has API for ecs.ContainerDefinition

https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ecs.ContainerDefinition.html

while Pulumi only provide a JSON string.

I am using .NET right now so cant use `pulumi-awsx`.

So to summarize, would like to see some sort of overview plan are for `pulumi-aws`, `pulumi-awsx`, and `cdk` going forward.

**Comment by stack72** [comment]
Hi @koliyo 

We have started the work to make AWSX work for .NET - that work is being tracked https://github.com/pulumi/pulumi-awsx/issues/308 so I am going to close this specific issue out so we can have it all centrally linked

Sorry for the timelapse here in not giving a response

Paul
